documentation/book/security.adoc

```diff
@@ -32,23 +36,34 @@ The public key is used by the Kafka brokers to verify the identity of Kafka clie
 Contains all the Kafka broker private and public keys (certificates signed with the cluster CA).
 `<cluster-name>-zookeeper-nodes`::
 Contains all the Zookeeper node private and public keys (certificates signed with the cluster CA).
+`<cluster-name>-topic-operator-certs`::
+Contains the private and public keys (certificates signed with the cluster CA) used for encrypting communication between the Topic Operator and Kafka / Zookeeper.
 
 All the keys are 2048 bits in size and are valid for 365 days from initial generation.
 
 NOTE: "certificates rotation" for generating new ones on their expiration will be supported in future releases.
 
-=== Kafka Listeners
+=== Interbroker Kafka Communication
 
-The data encryption is provided on two different listeners exposed by each Kafka broker.
+Communication between brokers is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
-Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
+=== Internodal Zookeeper Communication
 
-Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
+By deploying an stunnel sidecar within every Zookeeper pod, the Cluster Operator is able to provide data encryption and authentication between Zookeeper nodes in a cluster.
+The stunnel sidecar proxies all Zookeeper traffic, TLS decrypting data upon entry into a Zookeeper pod and TLS encrypting data upon departure from a Zookeeper pod.
+This TLS encrypting stunnel proxy is instantiated from the `spec.zookeeper.stunnelImage` specified in the Kafka resource.
 
-NOTE: You can use the `CLIENT` listener on port 9092 for unencrypted communication with clients.
+WARNING: Currently, the Cluster Operator only encrypts traffic between Zookeeper nodes, through Zookeeper's election port, 3888, and follower port, 2888.
+Traffic going through Zookeeper's client port, 2181, is not encrypted.  
 
 === Kafka Client connections via TLS
 
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
```

Comment:
I think this sentence can be deleted as it talks only about `REPLICATION` and this is now clients section.